terraform/environments/digital-prison-reporting/modules/ec2/scripts/bootstrap_apps.sh

```diff
@@ -114,37 +119,41 @@ cp ./kubectl /usr/bin/kubectl
 
 mkdir -p /home/ssm-user/.kube
 
+# NOMIS
 ## Add Kubeconfig
 cat <<EOF > $kubeconfig
 apiVersion: v1
 clusters:
 - cluster:
-    certificate-authority-data: $cp_k8s_cert_auth
-    server: $cp_k8s_server
-  name: $cp_k8s_cluster_name
+    certificate-authority-data: $nomis_cp_k8s_cert_auth
+    server: $nomis_cp_k8s_server
+  name: $nomis_cp_k8s_cluster_name
 contexts:
 - context:
-    cluster: $cp_k8s_cluster_name
+    cluster: $nomis_cp_k8s_cluster_name
     namespace: dpr-nomis-port-forwarder
     user: nomis-port-forwarder-migrated
-  name: $cp_k8s_cluster_name
-current-context: $cp_k8s_cluster_context
+  name: $nomis_cp_k8s_cluster_name
+current-context: $nomis_cp_k8s_cluster_context
 kind: Config
 preferences: {}
 users:
 - name: nomis-port-forwarder-migrated
   user:
-    token: $cp_k8s_cluster_token
+    token: $nomis_cp_k8s_cluster_token
 EOF
 
-# Configure Kubernetes Cluster for CP
+# NOMIS
 ## Permission for Config
 chmod 666 $kubeconfig
```

Comment:
Same suggestion as [above](https://github.com/ministryofjustice/modernisation-platform-environments/pull/6708/files#r1651139605) for renaming.